server/helpers/checkUser.js

```diff
@@ -16,7 +19,41 @@ class CheckUser {
                 }
 			}
 		)
-	}
+    }
+    
+    static loginCredentials (req,resp,next) {
+        db.query(
+			UserQuery.checkUserQuery(req.body.email),
+			(err,res) => {
+                if (res.rows.length < 1) {
+                    return resp.status(401).send({
+                        status:"error",
+                        message:"You have entered a wrong credential"
```

Comment:
Please use a better error message for this

Reply:
Alright